Domain Connect Spec Draft.adoc

```diff
@@ -1453,10 +1488,22 @@ is false.
 |Boolean
 |shared
 |(optional) 
-Indicates that the template is shared and the
-provider name can be passed in on the query string. If false, passing in the name on the 
-query string must have no impact. The default for this
-is false.
+This flag has been deprecated. It used to indicate that the template allowed a dynamic providerName
+on the query string. It is replaced with the sharedProviderName flag.
+
+|*Shared Provider Name*
+|Boolean
+|sharedProviderName
+|(optional) 
+This flag indicates that the template allows the caller to pass in additional information for the
+providerName. This information should augment the display of the providerName from the template.

```

Comment:
Shall we add a notice, to set also "shared" flag in the template for back-compatibility with DNS providers not yet in the spec 2.2?